### PR TITLE
css file for the waitList page

### PR DESCRIPTION
**Pull request by ainobs**

### CODE REVIEW

File: waitList/style.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+}
+.test {
+    width: 80%;
+    margin-left: 40px;
+}
+.text {
+    padding: 15px;
+    margin-left: 30px;
+    font-size: 20px;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
```
Comment (nwakaemecynthia) [inline waitList/style.css]: I will advice we all agree as to what font-family to use for uniformity.
Thank you so much.
This looks nice.

Can we also try handling the responsive view and add a snapshot of our work?

**Review by nwakaemecynthia — APPROVED** [review]

**Review by faithakinbobola — CHANGES_REQUESTED** [review]
your css file should not be in a folder. Do a "git pull" and add your code to the existing style.css file.
Take note of the font family and color also, it doesn't align with that of the homepage

**Comment by ainobs** [comment]
Screenshot (24)
Screenshot (25)

**Comment by ainobs** [comment]
Screenshot (27)

**Review by faithakinbobola — APPROVED** [review]

**Pull request merged**